packages/span-transformations/test/json/jaegerProtoConvertedToOTELReadableSpan.json

```diff
@@ -0,0 +1,44 @@
+        "http.target": "/todos/1",
+        "net.peer.name": "jsonplaceholder.typicode.com",
+        "net.peer.ip": "104.21.4.48",
+        "net.peer.port": 443,
+        "http.host": "jsonplaceholder.typicode.com:443",
+        "http.response_content_length_uncompressed": 83,
+        "http.status_code": 200,
+        "http.status_text": "OK",
+        "http.flavor": "1.1",
+        "net.transport": "ip_tcp",
+        "http.path": "/todos/1",
+        "http.request.headers": "{\"accept\":\"application/json, text/plain, */*\",\"user-agent\":\"axios/0.21.1\",\"traceparent\":\"00-096cb59a7500438fd7be37575fecb1fa-ca0ea597b91a47e5-01\",\"host\":\"jsonplaceholder.typicode.com\"}",
+        "http.response.headers": "{\"date\":\"Tue, 14 Dec 2021 09:51:02 GMT\",\"content-type\":\"application/json; charset=utf-8\",\"content-length\":\"83\",\"connection\":\"close\",\"x-powered-by\":\"Express\",\"x-ratelimit-limit\":\"1000\",\"x-ratelimit-remaining\":\"999\",\"x-ratelimit-reset\":\"1631494143\",\"vary\":\"Origin, Accept-Encoding\",\"access-control-allow-credentials\":\"true\",\"cache-control\":\"max-age=43200\",\"pragma\":\"no-cache\",\"expires\":\"-1\",\"x-content-type-options\":\"nosniff\",\"etag\":\"W/\\\"53-hfEnumeNh6YirfjyjaujcOPPT+s\\\"\",\"via\":\"1.1 vegur\",\"cf-cache-status\":\"HIT\",\"age\":\"7759\",\"accept-ranges\":\"bytes\",\"expect-ct\":\"max-age=604800, report-uri=\\\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\\\"\",\"report-to\":\"{\\\"endpoints\\\":[{\\\"url\\\":\\\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v3?s=9YGUNJFP76CJ7wZTcltDe7f9EuCOAL8OqOButi%2FRZdLWw9GkNAFLox42W9DoCxVGJb4HzHlLVZbrFBPLtRnPY3nMZLACx%2Bm9Bh8pX3UFe5MWt6lclQeaE0lEqKtvic29q0h%2Fsd1%2BG3KIjZmgbvBh\\\"}],\\\"group\\\":\\\"cf-nel\\\",\\\"max_age\\\":604800}\",\"nel\":\"{\\\"success_fraction\\\":0,\\\"report_to\\\":\\\"cf-nel\\\",\\\"max_age\\\":604800}\",\"server\":\"cloudflare\",\"cf-ray\":\"6bd688092ae9c286-FRA\",\"alt-svc\":\"h3=\\\":443\\\"; ma=86400, h3-29=\\\":443\\\"; ma=86400, h3-28=\\\":443\\\"; ma=86400, h3-27=\\\":443\\\"; ma=86400\"}",
+        "http.request.body": "",
+        "http.response.body": "{\n  \"userId\": 1,\n  \"id\": 1,\n  \"title\": \"delectus aut autem\",\n  \"completed\": false\n}",
+        "otel.status_code": "OK",
+        "span.kind": "client",
+        "service.name": "tomservice",
+        "telemetry.sdk.language": "nodejs",
+        "telemetry.sdk.name": "opentelemetry",
+        "telemetry.sdk.version": "1.0.1",
+        "otel.library.name": "@opentelemetry/instrumentation-http",
+        "otel.library.version": "0.27.0",
+        "internal.span.format": "proto"
+    },
+    "duration": [1639568422, 656647014],
+    "startTime": [3279040250, 254734993],
+    "endTime": [1639471828, 201999903],
+    "links": [],
+    "instrumentationLibrary": { "name": "@opentelemetry/instrumentation-http", "version": "0.27.0" },
+    "events": [],
+    "ended": true,
+    "status": {
+        "code": "OK"
```

Comment:
"OK" is not a valid value for the status code:
https://github.com/open-telemetry/opentelemetry-js-api/blob/633dbde83ffe62a6f54e0edb06b1fe083661a513/src/trace/status.ts#L26

```js
/**
 * An enumeration of status codes.
 */
export enum SpanStatusCode {
  /**
   * The default status.
   */
  UNSET = 0,
  /**
   * The operation has been validated by an Application developer or
   * Operator to have completed successfully.
   */
  OK = 1,
  /**
   * The operation contains an error.
   */
  ERROR = 2,
}

```

You need to covert the string to a proper enum

Reply:
My bad, fixed